src/components/Video/Video.tsx

```diff
@@ -127,23 +120,7 @@ const Video: React.FC<Props> = ({
           <CollapsibleText text={item.description} className={styles.description} maxHeight={isMobile ? 60 : 'none'} />
 
           <div className={styles.buttonBar}>
-            <Button
-              className={styles.bigButton}
-              color="primary"
-              variant="contained"
-              size="large"
-              label={startWatchingLabel}
-              startIcon={isEntitled ? <Play /> : undefined}
-              onClick={onStartWatchingClick}
-              active={play}
-              fullWidth={breakpoint < Breakpoint.md}
-            >
-              {progress ? (
-                <div className={styles.progressRail}>
-                  <div className={styles.progress} style={{ width: `${progress * 100}%` }} />
-                </div>
-              ) : null}
-            </Button>
+            {startWatchingButton}
```

Comment:
Can't you just put the `<StartWatchingButton >` component here directly? What's the point of passing it in as a prop?

Reply:
Because it's a container and `<Video />` is a component

Reply:
I don't really understand the difference in the convention then I guess. I would vote that the simplicity of just having the button in the jsx outweighs any sort of nuance with trying to limit the amount of logic inside of a component. Or is there something I'm missing?